### ISSUE TITLE
double-click and copy console

### ISSUE THREAD

**Issue by Alex-Jordan** [rbeezer/mathbook#261]
Research if there is a way to may console's html output such that doubleclicking (on reasonable browsers) highlights all of the input line, but not the prompt. Or perhaps some other way to make it epsilon easier to copy off the screen. See https://groups.google.com/d/msg/mathbook-xml-support/hesho0a0qfo/2GbAyJFEBgAJ

**Comment by Alex-Jordan** [comment]
Try this:
http://spot.pcc.edu/~ajordan/no-select.html

following this SO post:
http://stackoverflow.com/questions/826782/css-rule-to-disable-text-selection-highlighting

Since my first link may not be there forever, here is the content:

```
<!DOCTYPE html>

<style>
.unselectable {
  -webkit-touch-callout: none; /* iOS Safari */
  -webkit-user-select: none;   /* Chrome/Safari/Opera */
  -khtml-user-select: none;    /* Konqueror */
  -moz-user-select: none;      /* Firefox */
  -ms-user-select: none;       /* Internet Explorer/Edge */
  user-select: none;           /* Non-prefixed version, currently
                                  not supported by any browser */
}
</style>

<html>
<body>

<pre><span class="unselectable">alex.jordan$ </span>git status</pre>

</body>
</html>
```

**Comment by rbeezer** [comment]
That's wild!  I should be into consoles soon.

@davidfarmer: do you see any red flags here?  Can we add this style to the 
add-on?  I'd only write on the prompt of a console.

**Comment by Alex-Jordan** [comment]
One thing I'm not sure I like is that you *can't* copy the prompt part at all. Is there a use case for wanting to copy that part ever? Originally I was thinking something like:

1. put mouse over 'status'
2. double click to highlight 'status'
3. click a third time to highlight 'git status'
4. click a fourth time to highlight the whole line including the prompt

But it might be irritating to have to click just the right number of times to get the input part. So maybe this is better?

**Comment by rbeezer** [comment]
I can't imagine why you'd want the prompt.

I liked the way the demo behaved.

**Comment by davidfarmer** [comment]
I added this css to mathbook-add-on.css:

.mathbook-content .unselectable {  /* from Alex Jordan */
   -webkit-touch-callout: none; /* iOS Safari */
   -webkit-user-select: none;   /* Chrome/Safari/Opera */
   -khtml-user-select: none;    /* Konqueror */
   -moz-user-select: none;      /* Firefox */
   -ms-user-select: none;       /* Internet Explorer/Edge */
   user-select: none;           /* Non-prefixed version, currently
                                   not supported by any browser */
}

**Comment by rbeezer** [comment]
Implemented at addda923

Thanks, Alex and David - that's a nice one.
